apps/backend-e2e/src/maps.e2e-spec.ts

```diff
@@ -3170,7 +3170,7 @@ describe('Maps', () => {
         });
       });
 
-      afterAll(() => db.cleanup('mMap', 'user'));
+      afterAll(() => db.cleanup('mMap', 'user', 'mapReview'));
```

Comment:
You don't need to delete `mapReview` here since it's cascade deleted on `mMap` deletion. I've actually been meaning to create some tests for cascade deletions, though in practice we'd probably get E2E tests failing all over the place if one was missing (not really a good substitute for real tests, I know).